app/src/androidTest/java/org/kiwix/kiwixmobile/nav/destination/library/LibraryRobot.kt

```diff
@@ -0,0 +1,54 @@
+/*
+ * Kiwix Android
+ * Copyright (c) 2020 Kiwix <android.kiwix.org>
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package org.kiwix.kiwixmobile.nav.destination.library
+
+import applyWithViewHierarchyPrinting
+import org.kiwix.kiwixmobile.BaseRobot
+import org.kiwix.kiwixmobile.Findable
+import org.kiwix.kiwixmobile.R
+import org.kiwix.kiwixmobile.language.LanguageRobot
+import org.kiwix.kiwixmobile.language.language
+import org.kiwix.kiwixmobile.testutils.TestUtils
+
+/**
+ * Authored by Ayush Shrivastava on 29/10/20
+ */
+
+fun library(func: LibraryRobot.() -> Unit) = LibraryRobot().applyWithViewHierarchyPrinting(func)
+
+class LibraryRobot : BaseRobot() {
```

Comment:
these robots should be in separate files

Reply:
We merged online lib and local lib. that's why these two are together, so is the change still needed for moving 'em in different classes?

Reply:
OnlineLibrary is where we download items in `Download`, Library is where we view the files on our device in `Library`

These are 2 separate robots so they should be in their own files

Reply:
Fine